.storybook/recipes/DataSummaryCard/DataSummaryCard.tsx

```diff
@@ -0,0 +1,91 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './DataSummaryCard.module.css';
+
+import { Card, Heading, Text } from '../../../src';
+
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Main data text of the card.
+   */
+  data?: string;
+  /**
+   * Text that provides a unit of measurement for the data.
+   */
+  dataUnit?: string;
```

Comment:
Should I just make this `children: ReactNode`? Or just the data prop into children: `children?: ReactNode; dataUnit?: string`?

My justification for having this kind of api is that the two props `data` and `dataUnit` need to be styled differently

Reply:
I guess my question is whether we want this card to be versatile enough to accommodate other kinds of content. This feels like a very specific kind of card and that it makes sense for it to have a very specific api like this. I could also see the value of having a card that just has the band across the bottom and you can put whatever you want inside; I have a feeling we're going to see that pattern show up again somewhere else now that it's in the product. But for this card specifically, I think this is fine.

Reply:
I think my only concern is that these props might be a little hard to map to their usage. Maybe `data` could be `dataAmount` so it's clearer that it's the left half of the string thingy. I'm also wondering if `data` should be a `number` instead of a string and whether `dataUnit` should mention that it should start with a forward slash since we're not adding that.

Reply:
Hmmmm I think `dataAmount` makes sense but not sure if it should be a number since maybe there's a use case for displaying letter grades or something. There exists a use case for `dataUnit` to be a `%`, I'll change one of the stories to represent that.